src/flash-source-buffer.js

```diff
@@ -23,6 +25,17 @@ const scheduleTick = function(func) {
 };
 
 /**
+ * Generates a random string of length 10
+ *
+ * @return {String} the randomly generated string
+ * @function generateRandomString
+ * @private
+ */
+const generateRandomString = function() {
+  return (Math.random().toString(36) + '00000000000000000').slice(2, 8);
```

Comment:
Why add the zeros? Any value the string could take has a `1 / 2^52` chance of occurring - even the potential empty string. Also, why are there 17 zeros when you only take the first 10 characters that might be generated?